lib/pharos/phases/mixins/cluster_version.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Pharos
+  module Phases
+    module Mixins
+      module ClusterVersion
+        # @return [Gem::Version]
+        def existing_version
+          if cluster_version = cluster_context['existing-pharos-version']
```

Comment:
module assumes `cluster_context` method exists, but I guess it's a fair assumption(?)

Reply:
This mixin is intended to use only with Phase... so I think it's fair assumption.